cookiecutter/wrapper-template/{{cookiecutter.wrapper_id}}/{{cookiecutter.library_name}}/wrapper.py

```diff
@@ -0,0 +1,59 @@
+"""Meltano {{ cookiecutter.source_name }} extension."""
```

Comment:
@pandemicsyn - As I was thinking through the codebase here, I think we'd still want this file (which defines the subclass of `ExtensionBase` regardless of if the extension is wrapping another tool or if it is standalone. Can we rename this file to something like `extension_base.py` or `base.py` or `extension.py`?

Reply:
maaaybe, I was kinda wondering if we'd actually eventually split the entry point but thats easy enough to change later. I'll update it to extension.py. 👍